package.json

```diff
@@ -63,6 +63,7 @@
     "@commitlint/config-conventional": "17.4.4",
     "@justeat/browserslist-config-fozzie": "1.x",
     "@justeat/pie-design-tokens": "5.2.0",
+    "@justeattakeaway/generator-pie-component": "workspace:*",
```

Comment:
Could it be `pie-component-generator`?

Reply:
IIRC, the name has to start with `generator-` by convention